test/metabase/lib/breakout_test.cljc

```diff
@@ -214,7 +214,7 @@
 (deftest ^:parallel breakoutable-columns-source-card-test
   (doseq [varr [#'lib.tu/query-with-card-source-table
                 #'lib.tu/query-with-card-source-table-with-result-metadata]
-          :let [query (varr)]]
+          :let [query @varr]]
```

Comment:
I reworked the various queries in `lib.tu` to be plain defs rather than functions since there was really no reason they needed to be functions.